tests/appsec/iast/iast_fixtures.py

```diff
@@ -0,0 +1,46 @@
+from utils import weblog, interfaces, context
+
+
+def _get_expectation(d):
+    if d is None:
+        return None
+    if isinstance(d, str):
+        return d
+    expected = d.get(context.library.library)
+    if isinstance(expected, dict):
+        expected = expected.get(context.weblog_variant)
+    return expected
+
+
+class SinkFixture:
+    def __init__(
+        self, vulnerability_type, insecure_endpoint, secure_endpoint, data, location_map=None, evidence_map=None
+    ):
+        self.vulnerability_type = vulnerability_type
+        self.insecure_endpoint = insecure_endpoint
+        self.secure_endpoint = secure_endpoint
+        self.data = data
+        self.expected_location = _get_expectation(location_map)
+        self.expected_evidence = _get_expectation(evidence_map)
+        self.insecure_request = None
+        self.secure_request = None
+
+    def setup_insecure(self):
+        if self.insecure_request is None:
+            self.insecure_request = weblog.post(self.insecure_endpoint, data=self.data)
```

Comment:
If we expand this to sources we have to be careful since some of the operations use GETs instead of POSTs

Reply:
Yeah, I'll add a `method` argument once it's needed.